e2e/run.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+
+set -e
+
+# color codes
+RED='\033[0;31m'
+GREEN='\033[0;32m'
+YELLOW='\033[1;33m'
+BLUE='\033[0;34m'
+NC='\033[0m' # No Color
+
+function getPassboltSecret() {
+    local apiVersion="${1}"
+    kubectl get \
+        passboltsecrets.${apiVersion}.passbolt.tagesspiegel.de \
+        passboltsecret-sample-${apiVersion} \
+        -o json
+}
+
+function isSyncStatusSuccess() {
+    local jsonRsp="${1}"
+    local syncStatus=$(echo ${jsonRsp} | jq -r '.status.syncStatus')
```

Comment:
```suggestion
    local syncStatus
    syncStatus=$(echo "${jsonRsp}" | jq -r '.status.syncStatus')
```
split declaration and initialization to prevent masking the return code

Reply:
Not needed, since we use -x. I have now explicitly added this option to the script so that it is never masked.